WordCreate/WordCreate/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace WordCreate
+{
+    class Program
+    {
+        static void Main()
+        {
+            string word = "коронация";
+
+            string word1 = word.Substring(6, 1) + word.Substring(7, 1) + word.Substring(2,1) + word.Substring(0,1);
```

Comment:
Если вы берете из исходного слова по одной букве, то проще обращаться напрямую по индексу
```suggestion
            string word1 = "" + word[6] + word[7] + word[2] + word[0];
```